newsfragments/1234.significant.rst

```diff
@@ -0,0 +1,18 @@
+Since Airflow 2.9.0, following Listener API methods are not considered experimental anymore:
```

Comment:
```suggestion
Following Listener API methods are considered stable and can be used for production system (were experimental feature in older Airflow versions):
```

No need to mention the 2.9.0 version. It will be part of 2.9.0 changelog